frontend/src/app/users/[id]/page.tsx

```diff
@@ -0,0 +1,68 @@
+"use client";
+
+import { getUsersVideos, Progress, Video, ProgressObject } from "@/shared/api";
+import VideoCard from "@/shared/video-card";
+import Link from "next/link";
+import { useState, useEffect } from "react";
+
+export default function Home({ params }: { params: { id: string } }) {
+    const [videos, setVideos] = useState<Video[]>([]);
+    const [videoProgresses, setVideoProgresses] = useState<ProgressObject>({});
+
+    useEffect(() => {
+        const getVids = async () => {
+            const vids = await getUsersVideos(params.id);
```

Comment:
In a follow up PR could we had a meaningful message if there's no videos? A blank page isn't very reassuring